addon/macro.js

```diff
@@ -44,6 +87,6 @@ export default function createTranslatedComputedProperty(key, options) {
       );
     }
 
-    return intl.t(key, mapPropertiesByHash(this, hash));
+    return intl.t(key, Object.assign({}, staticValues, mapPropertiesByHash(this, dynamicValues)));
```

Comment:
`Object.assign` is not natively available in IE11. I guess we should use [`assign` from `@ember/polyfills`](https://emberjs.com/api/api/ember/release/functions/@ember%2Fpolyfills/assign) instead.